content/blog/2023/06/2023-06-20-membership-program.md

```diff
@@ -0,0 +1,150 @@
+
+[extra]
+image = "https://matrix.org/blog/img/matrix-logo.png"
++++
+
+We shared back in December how we wanted to find a way for people and
+organisations to support Matrix in a more impactful manner. We wanted it to also
+enable the Foundation to be more autonomous and more powerful in growing the
+ecosystem. Well the day has come: the Foundation is now able to formally accept
+members! 
+
+So what is a member of the Matrix.org Foundation? It’s an organisation or
+individual who financially supports the Foundation by paying a yearly membership
+fee, and in return gets some influence in driving the direction of Matrix and
+the Foundation’s activities.
+
+The key has been to create a balanced governance model, to make sure the
+Foundation stays aligned with the [Matrix manifesto](https://matrix.org/about/#matrix-manifesto).
+For this we’ve created a [Governing Board](https://matrix.org/membership/) to
+set the direction of the Foundation whilst the membership fees will fuel our
+efforts to get us there.
+
+So why become a member?
+
+## A common, durable platform
+
+Matrix is an open protocol everyone can contribute to. However we believe that
+to remain healthy, a protocol is better off having a single spec that needs to
+be curated. The additions need to be debated, implemented in actual software,
+and the rest of the specification needs to be adapted to the new changes. Such
+curation and edition work take time and effort, but the benefits of it are
+enormous.
+
+**A curated protocol benefits individuals, digital rights activists, and
+philanthropic investors who believe in the fundamental right to control and
+privacy in online communication**. As an open standard for real time
+communications, Matrix brings the sovereignty, security and interoperability,
+which are key to having full control over one’s own communication. By
+financially supporting the project, people who are aligned with our beliefs
+allow us to keep fighting for our collective rights, be it by providing secure
+software or persuading regulators to protect their citizens (such as our
+[DMA promotion work](https://matrix.org/blog/2023/03/15/the-dma-stakeholder-workshop-interoperability-between-messaging-services/)
+or our [fight against the Online Safety Bill](https://matrix.org/blog/2021/05/19/how-the-u-ks-online-safety-bill-threatens-matrix)).
+
+**A curated protocol benefits profit-making companies building communication
+products and services.** Decentralisation is a hard problem. End-to-end
+encryption is a hard problem. Decentralised end-to-end encrypted communications
+are a very hard problem. Matrix is doing the heavy lifting in these areas so
+companies can focus on building what they do best: creating great user
+experiences on top of it. By financially supporting the project, organisations
+building on Matrix rest assured it keeps evolving, being patched and staying
+secure. In short it ensures Matrix remains a competitive advantage over other
```

Comment:
"remainds advantaged" ("competitively"?) or, easier:

```suggestion
secure. In short it ensures Matrix maintains a competitive advantage over other
```

another alternative:

```suggestion
secure. In short it ensures Matrix keeps a competitive advantage over other
```

Reply:
I mean to say using Matrix is a competitive advantage, I'm not sure your suggestions keep that meaning

Reply:
No it doesn't, but it was not clear to me at all that that is what this was trying to say.